e2e/tests-dfx/bootstrap.bash

```diff
@@ -40,18 +40,3 @@ teardown() {
     assert_command_fail dfx bootstrap --port 8000
     assert_match "Cannot forward API calls to the same bootstrap server"
 }
-
-@test "bootstrap supports http requests" {
```

Comment:
Same with the other tests, can we use `dfx_new_frontend` rather than deleting tests?

In the future, if the intent is to disable a test to get the rest working, adding `skip` as the first line of a test will achieve this.